policyengine_us/parameters/gov/states/mt/tax/income/subtractions/subtractions.yaml

```diff
@@ -5,7 +5,6 @@ values:
   2021-01-01:
     - mt_interest_exemption_person
     - us_govt_interest
-    - taxable_social_security
     - mt_tuition_subtraction_person
```

Comment:
need to add the mt_taxable_social_security here right?

Reply:
mt_taxable_social_security can be either addition or subtraction (but not both), depending on its value. So I am thinking adding it directly to the mt_agi.

Reply:
Here's the rule
![image](https://github.com/PolicyEngine/policyengine-us/assets/113824411/093c6bf9-2a5a-49f5-9bec-ee5fcc4b7b7d)